.jshintrc

```diff
@@ -0,0 +1,74 @@
+{
+    "maxerr"        : 50,     //Max errors before it fails the build
```

Comment:
please remove '//' lines , you could refer to https://github.com/RackHD/on-http/blob/master/.jshintrc